### ISSUE TITLE
Log-Scale Histograms fill down to value

### ISSUE THREAD

**Issue by DomHin** [dcjones/Gadfly.jl#1567]
Hey developers,

I cannot imagine that I am the only one with this problem, but I have problems with Gadfly and `Geom.histogram` (same issue also with `Geom.bar` & `Geom.hair`)
In science everyone loves distributions and needs to display them in a comparable way. The best to do is just normalise them - this function works (`Geom.histogram(density=true`) nicely. 

However, make nice plots of histograms of any type with logarithmic scale does not work with Gadfly.
Here an example: 
```
x = randn(1000)
plot(x=x, Geom.histogram(density=true))
```
![density_hist](https://user-images.githubusercontent.com/23215094/151401497-a90a4203-4383-469e-91b9-5658e6f5ae1b.png)

BUT once you apply `Scale.y_log10` it looks like that: 
`plot(x=x, Geom.histogram(density=true), Scale.y_log10)`
![density_hist_log](https://user-images.githubusercontent.com/23215094/151401601-9f76a2bf-d1b0-41f0-aaa5-e8493e1bc133.png)

Obviously, it always fills until zero, which is for the log case 10e0.
The question is, is there a way to change a `fill_to` value for this? So that one can say that it should fill down to 10e-2 for example ?

Thanks for your help.

**Comment by bjarthur** [comment]
no, there isn't.  it seems kind of undefined to me.  perhaps it's best to plot points or lines instead of bars?

```
julia> x=randn(1000)
julia> plot(x=x, Stat.histogram(density=true), Geom.point, Scale.y_log10)
```
<img width="516" alt="Screen Shot 2022-01-28 at 5 53 33 PM" src="https://user-images.githubusercontent.com/1538268/151632703-040a9755-e0e9-4281-a494-40d2fa50ba47.png">
```
julia> plot(x=x, Stat.histogram(density=true), Geom.line, Scale.y_log10)
```
<img width="521" alt="Screen Shot 2022-01-28 at 5 53 42 PM" src="https://user-images.githubusercontent.com/1538268/151632721-212c2a3b-6842-4625-ab33-2649307bb324.png">

**Comment by DomHin** [comment]
Thanks for your comment @bjarthur 
Unfortunate, that there is no option to adjust this.
Some data must be drawn as histograms otherwise it could be misunderstood.
